modules/dxp/apps/portal-rules-engine-drools/portal-rules-engine-drools/bnd.bnd

```diff
@@ -3,35 +3,22 @@ Bundle-SymbolicName: com.liferay.portal.rules.engine.drools
 Bundle-Version: 5.0.9
 DynamicImport-Package: *
 Import-Package:\
-	!com.bea.*,\
```

Comment:
@rafaprax The commit message "Use exported xstream api" is confusing to me. So you add a separate bundle to expose "com.thoughtworks.xstream" packages, but the removed packages here are not the packages you exposed. 

If the lines you removed are actually useless, then please do a clean up before you made your changes.

Tina.